web/src/integTest/java/com/microsoft/applicationinsights/azuretest/TelemetryItem.java

```diff
@@ -0,0 +1,116 @@
+package com.microsoft.applicationinsights.azuretest;
+
+import java.util.ArrayList;
+import java.util.Hashtable;
+
+/**
+ * Created by moralt on 30/4/2015.
+ */
+public class TelemetryItem {
+    private static Hashtable<DocumentType, ArrayList<String>> defaultPropertiesToCompare = new Hashtable<DocumentType, ArrayList<String>>();
+    private DocumentType docType;
+    private String id;
+    private Hashtable<String, String> properties = new Hashtable<String, String>();
```

Comment:
You have a built in object called [Properties](http://docs.oracle.com/javase/7/docs/api/java/util/Properties.html) which is actually implemented as Hashtable, so no need to define one your own